### PR TITLE
Source GitHub: use new repo for SAT tests

### LINKED ISSUE #5721: Source GitHub: use `airbytehq/integration-test` repo for tests instead of `airbytehq/airbyte`
## Tell us about the problem you're trying to solve
Currently GitHub connector uses `airbytehq/airbyte` repo for tests. It eat up a lot of requests (a few thousand requests per each test run) and it's slow. In #4750 and #5674 repo for tests was created and filled with data. Now we need to use it in tests.

## Describe the solution you’d like
Update config with new repo and maybe update test files with state.

## Describe the alternative you’ve considered or used
Just live with slow tests and tons of requests ;(

## Additional context
Nope.

## Are you willing to submit a PR?
Yes.

### PR DESCRIPTION
**Pull request by Zirochkaa**
## What
Closes #5721.
Use new `airbytehq/integration-test` repo.
Remove unneeded files.

## Pre-merge Checklist

   
- [x] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret` 
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Code reviews completed
- [x] Documentation updated 
    - [x] Connector's `README.md`
    - [x] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [x] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)
- [ ] Connector version bumped like described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

### CODE REVIEW

**Review by avida — APPROVED** [review]

**Review by keu — APPROVED** [review]

**Review by yevhenii-ldv — APPROVED** [review]

File: airbyte-integrations/connectors/source-github/acceptance-test-config.yml [hunk 1]
```diff
@@ -11,30 +11,23 @@ tests:
...
       future_state_path: "integration_tests/abnormal_state.json"
       cursor_paths:
-        issues: ["airbytehq/airbyte", "updated_at"]
-        comments: ["airbytehq/airbyte", "updated_at"]
-        commits: ["airbytehq/airbyte", "created_at"]
-        commit_comments: ["airbytehq/airbyte", "updated_at"]
-        issue_milestones: ["airbytehq/airbyte", "updated_at"]
-        projects: ["airbytehq/airbyte", "updated_at"]
-        releases: ["airbytehq/airbyte", "created_at"]
+        events: ["airbytehq/integration-test", "created_at"]
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-github/acceptance-test-config.yml:20]: nice!

File: docs/integrations/sources/github.md [hunk 2]
```diff
@@ -86,6 +86,7 @@ Your token should have at least the `repo` scope. Depending on which streams you
 
 | Version | Date       | Pull Request | Subject |
 | :------ | :--------  | :-----       | :------ |
+| 0.1.8   | 2021-08-30 | [5726](https://github.com/airbytehq/airbyte/pull/5726) | Use `airbytehq/integration-test` repo for tests instead of `airbytehq/airbyte` |
```
Comment (sherifnada) [inline docs/integrations/sources/github.md:89]: we don't really need a version bump here since we're just updating tests

**Review by sherifnada — APPROVED** [review]

File: docs/integrations/sources/github.md (hunk 2 above)
Comment (Zirochkaa) [reply docs/integrations/sources/github.md:89]: Ok. Will remove it.

**Pull request merged**